fleet/fleet.go

```diff
@@ -234,6 +232,13 @@ func (f fleet) GetStatus(name string) (UnitStatus, error) {
 }
 
 func (f fleet) GetStatusWithExpression(exp *regexp.Regexp) ([]UnitStatus, error) {
+	status, err := f.GetStatusWithMatcher(exp.MatchString)
+	return status, maskAny(err)
+}
+
+// GetStatusWithMatcher returns a UnitStatus for each unit where the given matcher
```

Comment:
This is slightly misleading, if I understand correctly, as we can return `[]UnitStatus`, not just `UnitStatus`